src/Nethermind/Nethermind.sln

```diff
@@ -215,7 +215,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Nethermind.Serialization.Ss
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Nethermind.Network.Contract", "Nethermind.Network.Contract\Nethermind.Network.Contract.csproj", "{AD151E35-4BBC-4A83-8F57-CC8665F6E007}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Nethermind.SendBlobs", "Nethermind.SendBlobs\Nethermind.SendBlobs.csproj", "{E2CEB1D9-A81C-4142-9324-E086FF312636}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SendBlobs", "..\..\tools\SendBlobs\SendBlobs.csproj", "{EAA543AC-CD70-4C8E-925C-3E2D878ECA89}"
```

Comment:
Linking a project to the main solution that's not in the `src` directory is not a good approach. Either it moves to `src` or gets excluded from the solution. The latter may be.

Reply:
Removed the project from main solution